plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -1746,6 +1750,26 @@ public void testVacuum()
             assertThat(getActiveFiles(tableName)).isEqualTo(updatedFiles);
             // old files should be cleaned up
             assertThat(getAllDataFilesFromTableDirectory(tableName)).isEqualTo(updatedFiles);
+            // operations should be logged
+            assertThat(query("SELECT version, operation, MAP_FILTER(operation_parameters, (k, v) -> k <> 'queryId'), operation_metrics FROM \"" + tableName + "$history\"")).matches( """
+                    VALUES\s
+                    (CAST(0 AS BIGINT), CAST('CREATE TABLE AS SELECT' AS VARCHAR), CAST(MAP() AS MAP(VARCHAR, VARCHAR)), CAST(MAP() AS MAP(VARCHAR, VARCHAR))),\s
+                    (1, 'MERGE', MAP(), MAP()),\s
+                    (2, 'VACUUM START', MAP(), MAP(ARRAY['sizeOfDataToDelete', 'numFilesToDelete'], ARRAY['0', '0'])),\s
+                    (3, 'VACUUM END', MAP(ARRAY['status'], ARRAY['COMPLETED']), MAP(ARRAY['numDeletedFiles', 'numVacuumedDirectories'], ARRAY['0', '1'])),\s
```

Comment:
Why `numVacuumedDirectories ` is one if there are not deletedFiles?

Reply:
See the delta code https://github.com/delta-io/delta/blob/68275d150db7980419836d9b891c61cddb986457/spark/src/main/scala/org/apache/spark/sql/delta/commands/VacuumCommand.scala#L343

It is basically all the directories of the Delta table. `val dirCounts = allFilesAndDirs.where(col("isDir")).count() + 1 // +1 for the base path`

Reply:
numVacuumedDirectories is the number of scanned directories within the table location.